frontend/taipy-gui/src/components/Taipy/Indicator.tsx

```diff
@@ -96,19 +96,21 @@ const Indicator = (props: IndicatorProps) => {
 
     return (
         <Tooltip title={hover || ""}>
-            <Slider
-                id={props.id}
-                className={className}
-                min={0}
-                max={100}
-                value={getValue(value === undefined ? defaultValue : value, min, max)}
-                disabled={true}
-                valueLabelDisplay="on"
-                valueLabelFormat={getLabel}
-                marks={marks}
-                orientation={horizontalOrientation ? undefined : "vertical"}
-                sx={sliderSx}
-            ></Slider>
+            <div>
```

Comment:
I think a span would have less impact of the display ?

Reply:
I tried with a span and an empy <> but the tooltip doesn't work. 

Do you mean a new test needs to be added to `Indicator.spec.tsx`?

Reply:
a simple div like this is not acceptable as it would change the display behavior of the indicator to block: can't put 2 in the same line.
It looks like a tooltip directly on a Mui Slider (as used in indicator) doesn't work because the slider uses the same technique as the tooltip to show it's own different tooltips.
A div with `inline` display behaves the same as a span ie shows the hover text only below the indicator.
I suppose then that a span is better than nothing but still not great ...
I wonder if we should keep the hover_text all together.
thoughts @FabienLelaquais ?

PS: it looks like the test in Indicator.spec.tsx don't pass anymore ?